src/mon/OSDMonitor.cc

```diff
@@ -7971,6 +7975,10 @@ int OSDMonitor::prepare_new_pool(string& name,
       pg_num_min) {
     pi->opts.set(pool_opts_t::PG_NUM_MIN, static_cast<int64_t>(pg_num_min));
   }
+  if (osdmap.require_osd_release >= ceph_release_t::nautilus &&
```

Comment:
this should be quincy.

Reply:
or, maybe we could drop this entirely? i don't remember if the pool_opts are backward compatible...

Reply:
we need this for pacific if we want the new behavior there

Reply:
we need to verify what happens if you define a new pool_opts item.  Can older versions pass it along without knowing what it means?

Reply:
it looks ilke it'd be ok, the nautilus check appears to only matter for being able to encode int64 pool options.

We'd find out from an upgrade test with this PR pretty quickly.